drivers/include/periph/i2c.h

```diff
@@ -215,6 +215,26 @@ typedef enum {
  */
 void i2c_init(i2c_t dev);
 
+/**
+ * @brief   Change the pins of the given I2C bus back to plain GPIO functionality
+ *
+ * The pin mux of the SDA and SCL pins of the bus will be changed back to
+ * GPIO mode and the I2C bus is powered off.
```

Comment:
Can you clarify what GPIO mode (if it saves the previous state, has some default state, input, with or without pullups, etc.)?

Reply:
Depends no the implementation, `gpio_init()` should be called if you want a  defined state - I'll add a line.

Reply:
Depends no the implementation, `gpio_init()` should be called if you want a  defined state - I'll add a line.